DeflateWriter.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace samdark\sitemap;
+
+/**
+ * Flushes buffer into file with incremental deflating data, available in PHP 7.0+
+ */
+class DeflateWriter implements WriterInterface
+{
+    /**
+     * @var resource for target file
+     */
+    private $file = null;
```

Comment:
No need for `= null`, it's default.

Reply:
Oh right. I confused that with something else. E.g. I had to replace `unset($this->writerBackend)` with `$this->writerBackend = null`.